packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/OpenShiftVirtualMachinesRow.tsx

```diff
@@ -13,21 +12,15 @@ import { getVmTemplate } from './utils';
 const toNamespace = ({ data }: VMCellProps) =>
   (data.vm.providerType === 'openshift' && data.vm.object?.metadata?.namespace) || '';
 
-const toGVK = ({ data }) =>
-  (data.vm.providerType === 'openshift' && groupVersionKindForObj(data.vm.object)) || {
-    version: '',
-    kind: '',
-  };
-
 const cellRenderers: Record<string, React.FC<VMCellProps>> = {
   name: withResourceLink({
     toName: ({ data }) => data.name,
     toNamespace,
-    toGVK,
+    toGVK: () => ({ kind: 'VirtualMachine', version: 'v1', group: 'kubevirt.io' }),
   }),
   possibly_remote_namespace: withResourceLink({
     toName: toNamespace,
-    toGVK: () => ({ kind: 'Namespace', version: 'v1', group: 'core' }),
+    toGVK: () => ({ kind: 'Namespace', version: 'v1', group: '' }),
```

Comment:
right ...the group changed in newest API. I've checked against [older version ](https://docs.openshift.com/container-platform/3.11/rest_api/core/namespace-core-v1.html)